### DISCUSSION TITLE
Is there pre-trained Chinese-English Speech-to-Text Translation Models?

### DISCUSSION THREAD

**Discussion by duj12** [PaddlePaddle/PaddleSpeech#1234] [Q&A]
En-Zh ST model is released, i was wondering if there is Zh-En ST models too.

**Comment by zh794390558** [comment]
We do not have zh-en corpus.

**Reply by duj12** [reply]
I'm a totally freshman in ST, and there is non Zh-En E2E Speech-to-Text Translation benchmark, so I am confused about the performance of Zh-En ST tasks.
I noticed that Baidu has officially provided BSTC dataset,
https://aistudio.baidu.com/aistudio/competition/detail/44/0/datasets, which include about 68hour Zh-En ST data.
(Facebook also released Covost2 including about 26hours Zh-En ST data.)
is it possible to build a benchmark in Zh-En ST using BSTC dataset?
